### DISCUSSION TITLE
AudioSync when App is not open

### DISCUSSION THREAD

**Discussion by RobiStack64** [advplyr/audiobookshelf#2095] [General]
I could not find a previous discussion on this specific topic, although it seems likely one exists.

This issue was discovered on windows, running version 3.7 from Docker. I have encountered this with downloaded books, and have not yet tested it with streaming.

Currently, Audiobookshelf on mobile devices appears to update progress in an Audiobook exclusively when the app is open. If a user pauses and plays from a Bluetooth device, or uses the widget on their phone to pause and play, the next time they open the application, their timestamp will reset to whatever it was the last time it was fully open on their device. The only way to avoid this is to open the application while the book is playing to ensure the time stamp is updated before pausing. This causes various issues and renders things like sleep timers unusable.

If possible, I'd like to request an update be made so the time stamp reflects listening progress even when the app is not open on the phone, or both time stamps are tracked so the user can select whether to return to the old one or use the new one.

**Comment by nichwall** [comment]
This is a known issue with the bug fix releasing this weekend.
https://github.com/advplyr/audiobookshelf-app/issues/828
https://github.com/advplyr/audiobookshelf-app/issues/808
https://github.com/advplyr/audiobookshelf-app/issues/863

**Reply by RobiStack64** [reply]
Understood, thank you for the response
